### ISSUE TITLE
iPhone Mirroring with macOS Sequoia 

### ISSUE THREAD

**Issue by phpdave11** [pilotmoon/Scroll-Reverser#184]
I'm testing out macOS Sequoia Beta 2, and I noticed that the scroll direction is reversed in the new iPhone Mirroring app.

**Comment by pilotmoon** [comment]
Can you provide a bit more detail on this. Should it not reverse it? Since the purpose of Scroll Reverser is to reverse scrolling, a report that it is doing exactly that needs some extra context.

**Comment by phpdave11** [comment]
I have "Reverse Vertical" and "Reverse Mouse" enabled.

Using the trackpad, it scrolls as expected, the same way it would scroll when using the iPhone directly.  However, when using a mouse with a scroll wheel, it scrolls the opposite way.

For example, if I open Safari on macOS and scroll with the mouse wheel, it scrolls correctly based on how I have Scroll Reverser configured.  If I open Safari on iOS via iPhone Mirroring, it scrolls the opposite direction.

**Comment by fengvir** [comment]
Yes. the reverse vertical functions not working in iPhone Mirroring.

**Comment by DiaMeoww** [comment]
Need update for iPhone Mirroring

**Comment by tylxm** [comment]
Yes, iPhone Mirroring not working

**Comment by chenzz** [comment]
+1，iPhone Mirroring not working

**Comment by wamfox** [comment]
1+ scroll direction is wrong in the iPhone mirroring app.

**Comment by pilotmoon** [comment]
iPhone Mirroring app takes gestures as its input, instead of scrolling signals.

Scroll Reverser can reverse scrolling signal but not gestures.

For scroll wheels, it seems to be taking its input from deeper in the event stream than Scroll Reverser can access.

Therefore, unfortunately this is a can't fix. However, if some cleverer person than me can solve it I will be happy to receive a PR.

**Comment by DiaMeoww** [comment]
I use logic master 3 and use Logi Options+ could work on IPhone Mirroring, i dont konw how logic options works.

**Comment by pilotmoon** [comment]
I wonder if the Logi software uses a driver, perhaps. So as to work at a lower level.
